### PR TITLE
Powercord -> Replugged

### PR DESCRIPTION
**Pull request by asportnoy**
This should rename user-facing instances of "Powercord" to "Replugged". I left most internal stuff since many plugins might depend on them and could cause issues if they were renamed. In the future, we should gradually go through those and add replacements for backwards compatibility where needed.

### CODE REVIEW

**Review by Iucism — APPROVED** [review]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # READ FIRST
-This project has reached EOL, which means that no new features will be developed. We're only merging bug fixes and/or security pull requests. You can read more about this decision by joining our [official Discord server](https://discord.gg/powercord).
+This project has reached EOL, which means that no new features will be developed. We're only merging bug fixes and/or security pull requests. You can read more about this decision by joining our [official Discord server](https://discord.gg/replugged).
```
Comment (lleyton) [inline README.md]: I wish :(

File: injectors/linux.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const KnownLinuxPaths = Object.freeze({
     '/usr/lib64/discord-canary',
     '/opt/discord-canary',
     '/opt/DiscordCanary',
-    `${homedir}/.local/bin/DiscordCanary` // https://github.com/powercord-org/powercord/pull/370
+    `${homedir}/.local/bin/DiscordCanary` // https://github.com/replugged-org/replugged/pull/370
```
Comment (lleyton) [inline injectors/linux.js:30]: we should prob reference the old pull

File: src/patcher.js [hunk 3]
```diff
@@ -12,16 +12,16 @@ const PatchedBrowserWindow = require('./browserWindow');
...
 
 let _patched = false;
 const appSetAppUserModelId = electron.app.setAppUserModelId;
 function setAppUserModelId (...args) {
   /*
    * once this has been called, we can assume squirrelUpdate is safe to require
    * as everything that needs to be initialized has been initialized
-   * see: https://github.com/powercord-org/powercord/issues/405
-   * see: https://github.com/powercord-org/powercord/issues/382
+   * see: https://github.com/replugged-org/replugged/issues/405
```
Comment (lleyton) [inline src/patcher.js:23]: same here, we should reference the powercord issue

**Review by lleyton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (asportnoy) [reply README.md]: This text has been fully removed so no longer an issue

**Review by lleyton — APPROVED** [review]
LGTM

**Pull request merged**